pkgs/top-level/all-packages.nix

```diff
@@ -17478,6 +17478,11 @@ in
     qt = qt4;
   };
 
+  yadm = callPackage ../applications/version-management/yadm {
+    git = gitMinimal;
+    inherit stdenv fetchurl bash;
```

Comment:
You don't need to inherit these, callPackage will provide them to yadm/default.nix